XenAdmin/Controls/NetworkingTab/NetworkList.cs

```diff
@@ -478,7 +478,7 @@ private static bool HasPhysicalNonBondNIC(XenAPI.Network network)
         {
             foreach (PIF pif in network.Connection.ResolveAll(network.PIFs))
             {
-                if (pif.IsPhysical() && !pif.IsBondNIC())
+                if (pif.IsPhysical() && !pif.IsBondNIC() && pif.sriov_logical_PIF_of.Count == 0)

```

Comment:
IsPhysical() already checks `pif.sriov_logical_PIF_of.Count == 0`, so I don't think we need to add it here again.